python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -664,12 +664,12 @@ def graph_asset(
 
     Args:
         name (Optional[str]): The name of the asset.  If not provided, defaults to the name of the
-            decorated function. The asset's name must be a valid name in dagster (ie only contains
-            letters, numbers, and _) and may not contain python reserved keywords.
+            decorated function. The asset's name must be a valid name in Dagster (ie only contains
+            letters, numbers, and underscore) and may not contain Python reserved keywords.
```

Comment:
Nit - this should be plural, so `underscores`

Reply:
not sure how much this matters, but i noticed that the docs for `@asset` say "letters, numbers, and _)". Should i update that to "letters, numbers, and underscores) for consistency? I wonder if there are other places in our docs that say the same thing